### PR TITLE
MWPW-137671 Add support for hollow buttons and non static links with different breakpoint themes

### PR DESCRIPTION
**Pull request by vgoodric**
The addition of breakpoints for themes (desktop-dark, tablet-light etc) does not support hollow buttons or non static links. This PR adds that support. View the before and after pages on desktop, tablet and mobile sizes

Resolves: [MWPW-137671](https://jira.corp.adobe.com/browse/MWPW-137671)

Also making sure that the new CSS does not break static links.  Since there are multiple ways to set static links, I made a few pages to test:
Official "after" with no static links: https://theme-breakpoint-hollow--milo--adobecom.hlx.page/drafts/vgoodric/marquees-no-static?martech=off
Static links set as a variant on each marquee block: https://theme-breakpoint-hollow--milo--adobecom.hlx.page/drafts/vgoodric/marquees-block-static?martech=off
Static links set on the section metadata style: https://theme-breakpoint-hollow--milo--adobecom.hlx.page/drafts/vgoodric/marquees-section-static?martech=off
Static links set on section metadata style and marquees are in a fragment: https://theme-breakpoint-hollow--milo--adobecom.hlx.page/drafts/vgoodric/marquees-section-static-fragment?martech=off

Before version of all pages above:
Official "before" with no static links: https://main--milo--adobecom.hlx.page/drafts/vgoodric/marquees-no-static?martech=off
Static links set as a variant on each marquee block: https://main--milo--adobecom.hlx.page/drafts/vgoodric/marquees-block-static?martech=off
Static links set on the section metadata style: https://main--milo--adobecom.hlx.page/drafts/vgoodric/marquees-section-static?martech=off
Static links set on section metadata style and marquees are in a fragment: https://main--milo--adobecom.hlx.page/drafts/vgoodric/marquees-section-static-fragment?martech=off

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/vgoodric/marquees-no-static?martech=off
- After: https://theme-breakpoint-hollow--milo--adobecom.hlx.page/drafts/vgoodric/marquees-no-static?martech=off

### CODE REVIEW

**Review by seanchoi-dev — APPROVED** [review]
Looks solid to me.

File: libs/blocks/marquee/marquee.css [hunk 1]
```diff
@@ -25,6 +25,29 @@
   color: var(--color-white);
 }
 
+
```
Comment (overmyheadandbody) [inline libs/blocks/marquee/marquee.css:28]: Just one new line is needed

**Review by overmyheadandbody — COMMENTED** [review]
Per the process, we should be having JIRA stories for all PRs.

Otherwise code looks good, although it's not great to repeat the exact same styles in different locations, but I get that this level of theme flexibility is a requirement.

**Review by Sartxi — APPROVED** [review]

**Comment by narcis-radu** [comment]
Hey @vgoodric - please link the JIRA ticket in the description.
https://github.com/adobecom/milo/wiki/Reviewing-PRs

**Comment by rgclayton** [comment]
I don't think we should be getting a black/dark outline on hover. 

image

**Review by rgclayton — DISMISSED** [review]
Hover state shows a black/dark outline in the marquee. I don't think this is accurate. 
Let me know if specs have changed.

**Comment by rgclayton** [comment]
I’m still seeing the a black outline around the button in the last marquee - “Acrobat’s got it”
Is this expected?

image


On the first marquee the hover on the button does not get a black outline
I altered the bg color to gray to better see.
image

**Comment by vgoodric** [comment]
@rgclayton I was comparing the button to the wrong thing.  You're right, the outline should be white. Fix uploaded.

**Comment by rgclayton** [comment]
I'm not seeing the background change color per breakpoint now.

4th marquee on https://theme-breakpoint-hollow--milo--adobecom.hlx.page/drafts/vgoodric/marquees-block-static?martech=off

image

**Comment by rgclayton** [comment]
> I'm not seeing the background change color per breakpoint now.
> 
> 4th marquee on https://theme-breakpoint-hollow--milo--adobecom.hlx.page/drafts/vgoodric/marquees-block-static?martech=off
> 
> image

Ok. I think we can disregard this comment. Appears to be a separate issue with background colors not being supported per breakpoints. 

@vgoodric it might be helpful for historical reasons to clean up your example pages to avoid any confusion.

**Review by rgclayton — APPROVED** [review]

**Pull request merged**